recipes/micro/meta.yaml

```diff
@@ -0,0 +1,71 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "micro" %}
+{% set version = "1.3.3" %}
+{% set sha256 = "329f746e4ee9edf244618dda4208b638fda34c593d5cd96d8f71dc3b53e3d994" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+#  fn: {{ name }}-{{ version }}.tar.g
+  url: https://github.com/zyedidia/micro/releases/download/v1.3.3/micro-{{ version }}-linux64.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  # Uncomment the following line if the package is pure python and the recipe is exactly the same for all platforms.
+  # See https://conda-forge.org/docs/meta.html#building-noarch-packages for more details.
+  # noarch: python
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  skip: False
+#  script: python setup.py install --single-version-externally-managed --record record.txt
+
+#requirements:
+ # build:
+ #   - python
+    # When setuptools is available add the `--single-version-externally-managed --record record.txt` above.
+ #   - setuptools
+    # if your project compiles code (such as a C extension) then add `toolchain` as a build requirement.
+ # run:
+  #  - python
+
+test:
+  # Some package might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
+  commands:
+    - python --version
```

Comment:
You need to test your actual package. https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-section

Reply:
Hi @synapticarbors 

Now with this commit https://github.com/conda-forge/staged-recipes/pull/4291/commits/792e5d79a598b2e0036714d804786260356f1b9b  only the tests remains to be fixed for the package to be successfully done on CircleCI

Here's the CircleCI build log 
https://circleci.com/gh/conda-forge/staged-recipes/21129#build-parameters/containers/0